src/optimize.jl

```diff
@@ -161,7 +161,7 @@ function optimize!(code::CodeInfo, scope)
             # Check for :llvmcall
             arg1 = stmt.args[1]
             if (arg1 == :llvmcall || lookup_stmt(code.code, arg1) == Base.llvmcall) && isempty(sparams) && scope isa Method
-                uuid = uuid4()
+                uuid = uuid1()
```

Comment:
Maybe use an internal rng such that random numbers are the same for the user?
```suggestion
                uuid = uuid1(rng)
```

Reply:
What's the advantage?

Reply:
You get different random numbers if you interpret or not?

Reply:
Yeah that's a good idea.